packages/dds-components/src/components/InputStepper/InputStepper.stories.tsx

```diff
@@ -0,0 +1,80 @@
+import { type Meta, type StoryObj } from '@storybook/react';
+
+import { InputStepper } from './InputStepper';
+import { StoryHStack, StoryVStack } from '../Stack/utils';
+
+export default {
+  title: 'dds-components/InputStepper',
+  component: InputStepper,
+  argTypes: {
+    label: { control: 'text' },
+    decreaseButtonLabel: { control: 'text' },
+    increaseButtonLabel: { control: 'text' },
+    maxValue: { control: 'number' },
+    minValue: { control: 'number' },
+    step: { control: 'number' },
+    errorMessage: { control: 'text' },
+    htmlProps: { control: false },
+  },
+
+  parameters: {
+    docs: {
+      story: { inline: true },
+      canvas: { sourceState: 'hidden' },
+    },
+  },
+} satisfies Meta<typeof InputStepper>;
+
+type Story = StoryObj<typeof InputStepper>;
+
+export const Default: Story = {
+  render: args => {
+    return <InputStepper {...args} label="Label" />;
+  },
+};
+
+export const Overview: Story = {
```

Comment:
Split denne story i to; den ene med bare Sizes, og den andre med `errorMessage` og `tip`.